charts/k8s-image-swapper/templates/cert-manager-cert.yaml

```diff
@@ -7,8 +7,11 @@ metadata:
     {{- include "k8s-image-swapper.labels" . | nindent 4 }}
 spec:
   secretName: {{ include "k8s-image-swapper.fullname" . }}-cert
+  commonName: {{ printf "%s.%s.svc" (include "k8s-image-swapper.fullname" .) .Release.Namespace }}
   dnsNames:
-  - {{ include "k8s-image-swapper.fullname" . }}.{{ .Release.Namespace }}.svc
+    - {{ printf "%s.%s.svc" (include "k8s-image-swapper.fullname" .) .Release.Namespace }}
+    - {{ printf "%s.%s" (include "k8s-image-swapper.fullname" .) .Release.Namespace }}
+    - {{ include "k8s-image-swapper.fullname" . }}
```

Comment:
I see. Have you experienced issues in any of your clusters with this or is this mainly to get the tests working?

If it's solely for the tests I would prefer finding a solution not impacting how its currently working, e.g. only enable above options for testing? 🤔

Reply:
This shouldn't have any impact.  The certificate will still be issued to the same address it was before.  It will also have SANs (Subject Alternative Name) for the other valid forms of the address.  Where I could see this mattering is with metrics gathering.  Depending on what namespace the metrics monitoring software is running in and the method it uses for discovery or the address provided to it by a user configuring it, any of those addresses could be valid.  I certainly could wrap it in an if block to make it optional if you would prefer that.